core/controllers/base_test.py

```diff
@@ -269,6 +271,31 @@ def test_i18n_keys(self):
                     self.log_line('- %s' % key)
                 self.log_line('')
 
+    def alphabetic_test_i18n_keys(self):
+        """Tests that all JSON files are arranged in aplhabetical order."""
+        filenames = os.listdir(
+            os.path.join(os.getcwd(), self.get_static_asset_filepath(),
+                         'assets', 'i18n'))
+        for filename in filenames:
+            json_file_content = json.loads(utils.get_file_contents(
```

Comment:
Hi @billfinch -- this seems quite convoluted, because you're reading the file as a json object, printing it again, and then normalizing the spaces, and matching it with the original file. A simpler way to do this might be to open the original file as a text file, and check that each line follows the previous lexicographically (ignoring the first and last lines). It's ok not to normalize the spaces, because the file is supposed to be properly indented.

For example,
```
with open(filename, mode='rb') as f:
```